### ISSUE TITLE
seems to run out pixel cache when processing many images

### ISSUE THREAD

**Issue by sseiichi** [dlemstra/Magick.NET#194]
My program goes very slow after processed certain numbers of images in a for-loop.
This happens when Magick.Net starts writing files to temporary disk.
It seems Magick.Net depletes memory, but "using statement" and garbage collection seems to work well.

Here is my code.
```
            MagickNET.SetTempDirectory(@"C:\SomeTempFolder");

            for (int i = 0; i  < 10000; i++)
            {
                using (MagickImage image = new MagickImage(@"LargeImage.jpg"))
                {
                    image.Resize(new Percentage(50d));
                }
                Console.WriteLine(i); // At some point, this program goes slow.
            }
```

**Comment by dlemstra** [comment]
A similar issue has been reported here: https://www.imagemagick.org/discourse-server/viewtopic.php?f=27&t=33811 and a fix will be available soon.

**Comment by dlemstra** [comment]
@sseiichi Can you give it another try with the latest release?

**Comment by sseiichi** [comment]
@dlemstra It's solved now with version 7.4.4. Thanks a lot.
